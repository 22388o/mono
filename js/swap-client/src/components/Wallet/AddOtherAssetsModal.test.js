import React from 'react';
import { render, screen, waitFor } from '@testing-library/react';
import '@testing-library/jest-dom'; // For the "toBeInTheDocument" matcher
import { AddOtherAssetsModal } from './AddOtherAssetsModal';

describe('AddOtherAssetsModal component test', () => {
  it('renders component', async () => {

    const {container} = render(
      <AddOtherAssetsModal open={true} handleClose={() => {}} />
    );

    await waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });
});
