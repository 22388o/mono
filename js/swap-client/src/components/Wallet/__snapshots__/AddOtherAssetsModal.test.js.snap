// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddOtherAssetsModal component test renders component 1`] = `
<div
  aria-hidden="true"
/>
`;
